src/styles/quiz-card/quiz-card.css

```diff
@@ -0,0 +1,75 @@
+.quiz-card {
+  border-radius: 5px;
+  border: black 2px solid;
+  display: grid;
+  grid-template-columns: 16px min(40ch, calc(100% - 32px)) 16px;
```

Comment:
Do you need to set columns?

Reply:
right, in that case i can also solve it via the margin. I did that initial to be flexible in case an element should break out of the box.